src/Config/ConfigOverrides.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Drupal\stanford_profile_helper\Config;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\State\StateInterface;
+use Drupal\config_pages\ConfigPagesLoaderServiceInterface;
+use Drupal\config_pages\ConfigPagesInterface;
+use Drupal\Core\StreamWrapper\StreamWrapperManagerInterface;
+
+/**
+ * Config overrides for stanford profile.
+ *
+ * @package Drupal\stanford_profile_helper\Config
+ */
+class ConfigOverrides implements ConfigFactoryOverrideInterface {
+
+  /**
+   * State service.
+   *
+   * @var \Drupal\Core\State\StateInterface
+   */
+  protected $state;
+
+  /**
+   * Config pages loader service.
+   *
+   * @var \Drupal\config_pages\ConfigPagesLoaderServiceInterface
+   */
+  protected $configPagesLoader;
+
+  /**
+   * Config factory service.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * Config Entity Type Manager Interface.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * StreamWrapperInterface service.
+   *
+   * @var \Drupal\Core\StreamWrapper\StreamWrapperManagerInterface
+   */
+  protected $streamWrapperManager;
+
+  /**
+   * ConfigOverrides constructor.
+   *
+   * @param \Drupal\Core\State\StateInterface $state
+   *   State service.
+   * @param \Drupal\config_pages\ConfigPagesLoaderServiceInterface|null $config_pages_loader
+   *   Config pages service.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface|null $config_factory
+   *   Config factory service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface|null $entity_type_manager
+   *   Entity type manager interface.
+   * @param \Drupal\Core\StreamWrapper\StreamWrapperManagerInterface|null $stream_wrapper_manager
+   *   Stream wrapper manager interface.
+   */
+  public function __construct(
+    StateInterface $state,
+    ConfigPagesLoaderServiceInterface $config_pages_loader = NULL,
+    ConfigFactoryInterface $config_factory = NULL,
+    EntityTypeManagerInterface $entity_type_manager = NULL,
+    StreamWrapperManagerInterface $stream_wrapper_manager = NULL
+  ) {
+    $this->state = $state;
+
+    if ($config_pages_loader) {
+      $this->configPagesLoader = $config_pages_loader;
+    }
+
+    if ($config_factory) {
+      $this->configFactory = $config_factory;
+    }
+
+    if ($entity_type_manager) {
+      $this->entityTypeManager = $entity_type_manager;
+    }
+
+    if ($stream_wrapper_manager) {
+      $this->streamWrapperManager = $stream_wrapper_manager;
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+
+    // Theme settings override.
+    $this->setLockupOverrides($names, $overrides);
```

Comment:
So a user can change the lockup without changing the site name. That causes some issues because a user will change the lockup and expect their site to be indexed/displayed as something. When in fact the site name is still `University` and will be indexed like that. That'll produce bad things.

Reply:
This has to do with the fact that the original `line1` comes from the site name, which is overridden in another config form to start with. I talked to Ishi about defaulting line1 to the value in the other config form and cascading the changes but we figured it was better to leave it blank.

Reply:
Yeah its going to cause confusion I suspect. What if we just add some help text on the page saying something like "This changes the display of the lockup. But to change the actual site name, go here..."

Reply:
Cross-posted question here: https://stanfordwebservices.slack.com/archives/GJQJ7HTK7/p1595977283116100